seqlike/SeqLike.py

```diff
@@ -1069,7 +1069,7 @@ def swap_representation(s: SeqLike) -> SeqLike:
     return sc
 
 
-@dispatch(str, type(None), (str, Seq, SeqRecord, SeqLike))
+@dispatch(str, type(None), StringLikeType.__args__+ArrayType.__args__)
```

Comment:
the signature is likely intended to be consistent with line1084, and thus fixed here